AppUpdater.swift

```diff
@@ -183,9 +185,9 @@ extension Release: Comparable {
 }
 
 private extension Array where Element == Release {
-    func findViableUpdate(appVersion: Version, repo: String) throws -> Release.Asset? {
-        let properReleases = filter{ !$0.prerelease }
-        guard let latestRelease = properReleases.sorted().last else { return nil }
+    func findViableUpdate(appVersion: Version, repo: String, prerelease: Bool) throws -> Release.Asset? {
+        let suitableReleases = prerelease ? self : filter{ !$0.prerelease }
```

Comment:
Is this too concise at the expense of readability?

Reply:
I'm cool with it.